plugin/mouse/mouse.py

```diff
@@ -143,6 +143,13 @@ def mouse_drag_end():
         for button in buttons_held_down:
             ctrl.mouse_click(button=button, up=True)
 
+    def toggle_mouse_drag(button: int):
+        """If the button is held down, release all held buttons, else start dragging"""
+        if button in list(ctrl.mouse_buttons_down()):
+            self.mouse_drag_end()
```

Comment:
I wonder if we only should release the button specified in the `button` argument?
Something like `ctrl.mouse_click(button=button, up=True)`